packages/cf-component-tabs/src/Tabs.js

```diff
@@ -3,6 +3,17 @@ const { PropTypes } = React;
 const Viewport = require('cf-component-viewport');
 const Select = require('cf-component-select');
 
+const find = (list, condition) => {
+  let foundElement = undefined;
+  list.forEach(element => {
+    if (condition(element)) {
+      foundElement = element;
+      return;
```

Comment:
Good catch! I just removed the return since the list realistically isn't gonna be much bigger than 10 elements.